raiden/message_handler.py

```diff
@@ -28,12 +28,12 @@
     ReceiveSecretRequest,
     ReceiveSecretReveal,
     ReceiveTransferRefund,
-    ReceiveTransferRefundCancelRoute,
-    ReceiveSecretRequestLight, ReceiveSecretRevealLight)
+    ActionTransferReroute,
```

Comment:
Just to be clear, this event represents the refund received plus try another route? The side effects are the same as the old `ReceiveTransferRefundCancelRoute`

Reply:
It would be nice to explain that on the PR description or comments, just to know the reasons behind the refactor.

Reply:
Lets point to the raiden related issue too

Reply:
[Here's the PR fix in raiden](https://github.com/raiden-network/raiden/commit/1cc9eeccbdbb761cd68c6b8324a4c3a854516bb8) they sepparate the functionality in two parts, ActionTransferReroute and ReceiveTransferCancelRoute.
I'll edit the PR description to make a link to this fix